src/app/components/Metadata/utils/index.tsx

```diff
@@ -1,16 +1,27 @@
-import React from 'react';
+/* eslint-disable @typescript-eslint/ban-ts-comment */
+/** @jsx jsx */
```

Comment:
Similar here, `/** @jsxRuntime classic */` will need added above to support NextJS.